q2_sample_classifier/classify.py

```diff
@@ -246,6 +246,7 @@ def maturity_index(output_dir: str, table: biom.Table,
 
     # predict treatment data
     index = importances.index
+    table = _extract_features(table)
```

Comment:
Why wasn't this extraction happening already? Did this get nuked in the #100 business?

Reply:
see ln 232. Extraction was happening during `_load_data` previously — I disabled extraction there so that we can keep the table in biom format as long as possible (to maintain compatibility with changes to `split_table` in this PR, I believe)

Reply:
👍